### PR TITLE
Add without-libtirpc flag to configure

### PR DESCRIPTION
**Pull request by sac-urs**

### CODE REVIEW

File: tasks/prereq.yml [hunk 1]
```diff
@@ -70,8 +70,14 @@
   args:
     chdir: "{{ glusterfs_perf_git_dest | default('/usr/src/glusterfs') }}"
 
+- name: Configure without-libtirpc if version is set
+  set_fact:
+     wolibtirpc: "--without-libtirpc"
+  when: glusterfs_perf_git_version is not defined or
+        glusterfs_perf_git_version < 6
```
Comment (amarts) [inline tasks/prereq.yml]: generally it is a string, and in the form of v5.0 or v6.0 so not sure if this check is fine...

Comment (sac-urs) [reply tasks/prereq.yml]: Yeah, I realised it now, when I was testing. I'll see what I can do.

Comment (sac-urs) [reply tasks/prereq.yml]: @amarts I've pushed another commit, please check.

Comment (amarts) [reply tasks/prereq.yml]: For the current set of tags we need to test, this works. Thanks.

**Pull request merged**